app/services/bulk_edit_service.rb

```diff
@@ -8,11 +8,24 @@ def self.perform(collection_id:, attributes:, metadata_adapter: Valkyrie.config.
     c.decorate.members.each do |member|
       logger.info "Updating attributes for #{member}"
       change_set = ChangeSet.for(member)
-      if change_set.validate(attributes)
+      if change_set.validate(incorporate_proposed_attributes(member.attributes, attributes))
         change_set_persister.save(change_set: change_set)
       else
         logger.warn "  Failed validation: #{change_set.errors}"
       end
     end
   end
+
+  def self.incorporate_proposed_attributes(existing_attributes, proposed_attributes)
```

Comment:
I might call it something else like maybe "append_or_replace_attributes"?

Reply:
:+1: to the rename. It's more work, but I'd probably propose adding an `append_to_collection_id` virtual property similar to how we do `append_id`, so you can just set that property and we don't need special logic. I don't want to grind this work to a halt, though.

Reply:
Done